### ISSUE TITLE
Financial Services content accessibility audit

### ISSUE THREAD

**Issue by C13L0** [phsc-edu/2018#327]
@curtiskphsc Please review Financial Services for accessibility issues on some of their pages. There are many words and sentences in all caps. 

https://financial-services.phsc.edu/purchasing/purchase-order-terms-and-conditions

![financial-services-content-accessibility-issue](https://user-images.githubusercontent.com/3138019/43465812-2822cc9a-94ac-11e8-95b7-46b0e1b26888.png)

**Comment by curtiskphsc** [comment]
@C13L0 This is a legal document. I don't make or suggest edits to legal documents as I am not en attorney.

**Comment by C13L0** [comment]
@boettcc @curtiskphsc This is a webpage not a legal document and it is not ada compliant.

**Comment by curtiskphsc** [comment]
@C13L0 I will pass along your comments to the legal team and see what they say. It is a legal document that has been placed on a web page. Perhaps they need to remove it.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc 

The website is not a legal document, it is a website, meaning all pages must be ada compliant. As of right now, that page is not accessible to those using a screenreader because of content that is in all caps. The original EOE-ADA Compliance page was written in legal text but for this very reason was cited by OCR, along with 13 other pages.

To be clear, all pages on our main website and subdomains must be accessible and ADA compliant. I will bring this issue to the attention of Dr. Harts so she can be aware of potential OCR complaints.

**Comment by curtiskphsc** [comment]
Email sent to content owners with Github ticket URL attached. Awaiting comment.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc Please move forward with making the requested website content accessible. 

--------------

Good afternoon, Brian.

As we discussed, the caps on the page would have to be made lower case.  The page is not a legal document even though as you explained your office is using it as a legal reference.  As you know, we were sited for these very same issues less than a year ago.  

Now, if we were making the text into an electronic PDF, it would need to go through ADA compliant tools to be attached to the web page.  However, from our discussion this is not the case.  

Thanks for meeting.  

Melissa Harts, Ed.D
Vice President of Technology and Distance Education
Pasco-Hernando State College
10230 Ridge Road
New Port Richey, FL 34654
(727) 816-3466

**Comment by C13L0** [comment]
@curtiskphsc @boettcc Please correct the accessibility issues as outlined in the original request.

**Comment by C13L0** [comment]
@curtiskphsc @boettcc This page is still not compliant. Please make sure you are reviewing the content before publishing.

<img width="534" alt="ada" src="https://user-images.githubusercontent.com/3138019/44152094-8678cc0a-a072-11e8-854c-8eb59b83d25f.png">

**Comment by curtiskphsc** [comment]
This was referenced in another ticket and was repaired. closing.

**Comment by C13L0** [comment]
Completed
